packages/core-v2-typechain/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
+    "outDir": "dist/",
+    "target": "es5",
+    "module": "commonjs",
+    "declaration": true,
+    "esModuleInterop": true,
+    "moduleResolution": "node",
+    "strict": true,
+    "resolveJsonModule": true,
+    "skipLibCheck": true,
+    "paths": {
+      "src/*": [
+        "./src/*"
+      ],
+    }
+  },
+  "include": [
+    "./hardhat.config.ts"
```

Comment:
maybe add some comments that this file will be omitted from build

Reply:
Hmm, I think it's pretty clear since there is another tsconfig file right next to this one with "build" in it and the build npm script references the build config. If anything, I would have needed comments to explain the old way. This new way is just following convention as shown in the typescript docs: https://www.typescriptlang.org/docs/handbook/compiler-options.html#using-the-cli